paket.lock

```diff
@@ -0,0 +1,108 @@
+CONTENT: NONE
```

Comment:
What exactly is this file good for? Does it tell the exact version ranges needed? What is the difference to packet.dependencies?